esy-core/esy-build-package/BuildInfo.re

```diff
@@ -28,8 +28,8 @@ let read = (task: BuildTask.t) => {
         Ok(None);
       }
     );
-  switch read {
+  switch (read) {
   | Ok(v) => v
   | Error(_) => None
   };
-};
+};
```

Comment:
Why it removes newlines at the of files, I thought it is "correct" to have a new line at the of file cc @jordwalke

Reply:
I don't know what is "correct" and I never understood the inserted newline at end of file convention. Still we should make `refmt` add the newline because some editors add it every time you save, and so that would produce two file changes (once for the refmt, and once for the editor auto-inserting the newline).

Reply:
Yeah, at least on master, refmt should be placing the newlines. What editor do you use @rauanmayemir? What's the configuration for newlines at end of file?

Reply:
I checked VSCode configuration, even set up prettier to disable formatting for ocaml/reason to be safe, and newlines are still removed on save.

Reply:
How far ahead master is from `"@esy-ocaml/reason": "^3.0.5"`? (but vscode always did this iirc)

Reply:
Found the issue, that's vscode's doing.